dtfs-central-api/api-tests/v1/utilisation-reports/put-utilisation-report-status.api-test.ts

```diff
@@ -0,0 +1,304 @@
+      report,
+      status: reportStatus,
+    }));
+    const requestBody = {
+      user: MOCK_TFM_USER,
+      reportsWithStatus,
+    };
+
+    // Act
+    const { status } = await api.put(requestBody).to(setStatusUrl);
+    const updatedDocuments = await Promise.all(
+      reportsWithStatus.map((reportWithStatus) => utilisationReportsCollection?.findOne({ _id: new ObjectId(reportWithStatus.report.id) })),
+    );
+
+    // Assert
+    expect(status).toBe(200);
+    updatedDocuments.forEach((document) => expect(document?.status).toEqual(reportStatus));
+  });
+
+  it('returns a 200 if the request body only uses the report month, year and bank id', async () => {
+    // Arrange
+    const reportStatus = UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION;
+    const reportsWithStatus = uploadedReportDetails.map((report) => ({
+      report,
+      status: reportStatus,
+    }));
+    const requestBody = {
+      user: MOCK_TFM_USER,
+      reportsWithStatus,
+    };
+
+    // Act
+    const { status } = await api.put(requestBody).to(setStatusUrl);
+    const updatedDocuments = await Promise.all(
+      reportsWithStatus.map(({ report }) =>
+        utilisationReportsCollection?.findOne({
+          month: report.month,
+          year: report.year,
+          'bank.id': report.bankId,
+        }),
+      ),
+    );
+
+    // Assert
+    expect(status).toBe(200);
+    updatedDocuments.forEach((document) => expect(document?.status).toEqual(reportStatus));
+  });
+
+  it('returns a 200 if the request body has a combination of report identifiers', async () => {
+    // Arrange
+    const reportStatus = UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED;
+    const reportWithStatusWithBankId = {
+      report: uploadedReportDetails[0],
+      status: reportStatus,
+    };
+    const reportWithStatusWithReportId = {
+      report: uploadedReportIds[1],
+      status: reportStatus,
+    };
+    const reportsWithStatus = [reportWithStatusWithBankId, reportWithStatusWithReportId];
+    const requestBody = {
+      user: MOCK_TFM_USER,
+      reportsWithStatus,
+    };
+
+    // Act
+    const { status } = await api.put(requestBody).to(setStatusUrl);
+    const updatedDocuments = await Promise.all([
+      utilisationReportsCollection?.findOne({
+        month: reportWithStatusWithBankId.report.month,
+        year: reportWithStatusWithBankId.report.year,
+        'bank.id': reportWithStatusWithBankId.report.bankId,
+      }),
+      utilisationReportsCollection?.findOne({ _id: new ObjectId(reportWithStatusWithReportId.report.id) }),
+    ]);
+
+    // Assert
+    expect(status).toBe(200);
+    updatedDocuments.forEach((document) => expect(document?.status).toEqual(reportStatus));
+  });
+
+  describe('when the queried report does not already exist', () => {
+    it(`creates a new report with undefined azureFileInfo if the status to set is ${UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED}`, async () => {
```

Comment:
Valid point! Soon we are refactoring so that reports will always exist and we'll only be updating their status, this is just an intermediate solution while we are releasing private beta and dont have full functionality yet. I think post or put could be fine but given we'll be factoring soon to only be updated would prefer to keep this as PUT for now!